README.md

```diff
@@ -65,6 +65,58 @@ Note that you can also use asymmetric keys by using either the
 `PSR7Sessions\Storageless\Http\SessionMiddleware` constructor or the named
 constructor `PSR7Sessions\Storageless\Http\SessionMiddleware::fromAsymmetricKeyDefaults()`
 
+#### Session Hijacking mitigation
+
+To mitigate the risks associated to cookie stealing and thus
+[session hijacking](https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html#binding-the-session-id-to-other-user-properties),
+you can bind the user session to its IP (`$_SERVER['REMOTE_ADDR']`) and
+User-Agent (`$_SERVER['HTTP_USER_AGENT']`) by enabling client fingerprinting:
+
+```php
+use PSR7Sessions\Storageless\Http\ClientFingerprint\Configuration as FingerprintConfig;
+
+$app = new \Mezzio\Application(/* ... */);
+
+$app->pipe(new \PSR7Sessions\Storageless\Http\SessionMiddleware(
+    \Lcobucci\JWT\Configuration::forSymmetricSigner(
+        new \Lcobucci\JWT\Signer\Hmac\Sha256(),
+        \Lcobucci\JWT\Signer\Key\InMemory::base64Encoded('mBC5v1sOKVvbdEitdSBenu59nfNfhwkedkJVNabosTw='), // replace this with a key of your own (see docs below)
+    ),
+    \PSR7Sessions\Storageless\Http\SessionMiddleware::buildDefaultCookie(),
+    100,
+    \Lcobucci\Clock\SystemClock::fromSystemTimezone(),
+    fingerprintConfig: FingerprintConfig::forIpAndUserAgent()
```

Comment:
nnnnngggghhhh named arguments :| 

This highlights the smell that we have bad naming or too many arguments: the named argument should perhaps go away, before we merge

Reply:
Rather than "fingerprint config", this is the "session validation"?